### DISCUSSION TITLE
Invalid query using uuid

### DISCUSSION THREAD

**Discussion by CleitonDeLima** [codingjoe/django-select2#28] [General]
Hi @codingjoe ,

I encountered this [problem](https://github.com/carltongibson/django-filter/issues/1241) when using field select2 with django-filter, using parameters from a GET request.  
I analyzed the [code](https://github.com/codingjoe/django-select2/blob/master/django_select2/forms.py#L458) and saw what happens when rendering the form in error, an invalid query is made with UUID.

_Remember that the problem only happens when the GET parameter in the url is changed._

Can anything be done to resolve this?

Sorry for my English, it is not my native language.

**Comment by codingjoe** [comment]
Hi @CleitonDeLima,

Thanks for reaching out. Don't worry about your English, it is my second language too.

However, I am afraid I can't really follow you. The problem you are referring to, is on `django-filters` not `django-select`.
Maybe you can provide me with a little more information or maybe some example code or screenshots to showcase your problem.

Best,
Joe

**Comment by CleitonDeLima** [comment]
Hi @codingjoe 

I created [this](https://github.com/CleitonDeLima/django-select2/pull/1/) example to show the behavior that I commented on.
Removing the select2 widget [here](https://github.com/CleitonDeLima/django-select2/pull/1/files#diff-ebc3c23febf1da936edb4f180e004f8072466f4f48788fa2c8b8cb7267356038R234), the behavior is different. My question is whether this is something to be dealt with or not.

Errors can be seen in the CI

**Reply by codingjoe** [reply]
I still don't know if I can follow you, how is this related to this package? The test you wrote is an expected error triggered by Django. I don't know if we have any tests for UUID primary key support, but I know of a production project that uses UUID PKs and uses this package as well. There should be no issue.
Sorry, that I can't be more helpful, but I still don't really know how I can help better.
